crates/bevy_text/src/font_atlas_set.rs

```diff
@@ -76,10 +76,16 @@ impl FontAtlasSet {
             )
         };
         if !font_atlases.iter_mut().any(add_char_to_font_atlas) {
+            let glyph_max_size = glyph_texture
+                .texture_descriptor
+                .size
+                .height
+                .max(glyph_texture.texture_descriptor.size.width);
+            let containing = (1u32 << (32 - glyph_max_size.leading_zeros())).max(512) as f32;
```

Comment:
This needs a comment explaining what's being done in this line before I'm comfortable merging it.